docs/reference/repositories-metering/apis/get-repositories-metrics.asciidoc

```diff
@@ -0,0 +1,152 @@
+[[get-repositories-metrics-api]]
+=== Repositories metrics API
+++++
+<titleabbrev>Repositories metrics API</titleabbrev>
+++++
+
+Returns cluster repositories metrics.
+
+[[get-repositories-metrics-api-request]]
+==== {api-request-title}
+
+`GET /_nodes/<node_id>/_repositories_metrics`
+
+[[get-repositories-metrics-api-desc]]
+==== {api-description-title}
+
+You can use the cluster repositories metrics API to retrieve repositories metrics in a cluster.
+
+
+All the nodes selective options are explained <<cluster-nodes,here>>.
+
+[[get-repositories-metrics-api-path-params]]
+==== {api-path-parms-title}
+
+include::{es-repo-dir}/rest-api/common-parms.asciidoc[tag=node-id]
+
+[[get-repositories-metrics-api-query-params]]
+==== {api-query-parms-title}
+
+[role="child_attributes"]
+[[get-repositories-metrics-api-response-body]]
+==== {api-response-body-title}
+
+`_nodes`::
+(object)
+Contains statistics about the number of nodes selected by the request.
++
+.Properties of `_nodes`
+[%collapsible%open]
+====
+`total`::
+(integer)
+Total number of nodes selected by the request.
+
+`successful`::
+(integer)
+Number of nodes that responded successfully to the request.
+
+`failed`::
+(integer)
+Number of nodes that rejected the request or failed to respond. If this value
+is not `0`, a reason for the rejection or failure is included in the response.
+====
+
+`cluster_name`::
+(string)
+Name of the cluster. Based on the <<cluster.name>> setting.
+
+`nodes`::
+(object)
+Contains repositories metrics for the nodes selected by the request.
++
+.Properties of `nodes`
+[%collapsible%open]
+====
+`<node_id>`::
+(array)
+An array of repository metrics for the node.
++
+.Properties of objects in `node_id`
+[%collapsible%open]
+=====
+`repository_name`::
+(string) 
+Repository name.
+
+`repository_type`::
+(string) 
+Repository type.
+
+`repository_location`::
+(string) 
+Represents an unique location within the repository, 
+i.e. a `bucket` + `base_path`.
+
+`repository_ephemeral_id`::
+(string) 
+An identifier that changes every time the repository is updated.
+
+`repository_started_at`::
+(long)
+Time the repository was created or updated. Recorded in milliseconds
+since the https://en.wikipedia.org/wiki/Unix_time[Unix Epoch].
+
+`repository_stopped_at`::
+(long)
+Time the repository was deleted or updated. Recorded in milliseconds
+since the https://en.wikipedia.org/wiki/Unix_time[Unix Epoch].
+
+`archived`::
+(boolean) 
+A flag that tells whether or not this object has been archived.
+
+`cluster_version`::
+(long) 
+The cluster state version when this object was created, this field
+can be used as a logical timestamp to delete all the archived metrics up
+to a observed cluster version.
+
+`request_counts`::
+(object) 
+An object with the number of request performed against the repository
+grouped by request type.
++
+.Properties of `request_counts` for repository type `Azure`
+[%collapsible%open]
+======
+`HEAD`::
```

Comment:
I think we should rename these so that they more closely reflect what's being called in Azure. For example, this should be "GetBlobProperties", in order to avoid confusion with the HTTP verb (there is not always 1-to-1 correspondance)

Reply:
Also, ideally we should link to the pricing categorization page here (https://azure.microsoft.com/en-us/pricing/details/storage/blobs/)